WebIDL/ecmascript-binding/interface-prototype-object.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Interface prototype objects</title>
+<link rel="help" href="https://heycam.github.io/webidl/#interface-prototype-object">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+// A specification issue was raised for this behavior.
+// https://www.w3.org/Bugs/Public/show_bug.cgi?id=28244
+test(function() {
+    assert_class_string(Object.getPrototypeOf(Document.prototype),
+                        'DocumentPrototype');
```

Comment:
Added 3rd and 4th asserts.
As for 1st and 2nd, they are tests for toStringTag of an instance. Because this test file is for interface prototype object, they are out of scope.